framework/auth/oauth_scopes.py

```diff
@@ -71,6 +77,9 @@ class ComposedScopes(object):
     USERS_READ = (CoreScopes.USERS_READ,)
     USERS_WRITE = USERS_READ + (CoreScopes.USERS_WRITE,)
 
+    USERS_FULL_READ = USERS_READ + (CoreScopes.USERS_ADDON_READ,)
```

Comment:
Yes, the reason why we started implementing this was to allow a user to create a set of projects and add their google drive folders to each project based on where in the structure the project existed. There are also use cases that Jeff has requested which will need this capability for third party apps.

In reality, there's very little that we're likely to do with the API that won't be exposed to apps that want the capability.